index.js

```diff
@@ -2,10 +2,19 @@ const express = require('express');
 const dotenv = require('dotenv');
 const app = express();
 dotenv.config();
-const { startReportScan } = require('./reddit');
+const { KnicksRedditBot } = require('./reddit');
 
 const port = process.env.PORT || 5000;
 
+//Bot setup
+const clientId = process.env.CLIENT_ID;
+const clientSecret = process.env.CLIENT_SECRET;
+const refreshToken = process.env.REFRESH_TOKEN;
+const subreddit = process.env.SUBREDDIT;
+const badSourceFlairID = process.env.BAD_SOURCE_FLAIR_ID;
+
+bot = new KnicksRedditBot(clientId, clientSecret, refreshToken, subreddit)
```

Comment:
1) I'd never ever define a variable without let/const. The default should be const in JS as well.

2) Is this being used again anywhere? Or is the plan to use it again in the future?

Reply:
1. Forgot to use const -- fixed
2. Essentially it is the entire program as when its instantiated it starts polling and runs all the checking.
Wasn't really too sure how to implement it.

Reply:
If it's going to be a reused variable, I'd keep it in the running app.listen. If for some reason app crashed, and the bot was still running, that'd be bad and you'd have no way to kill it (I guess I'm throwing an edge case out there)

Reply:
Moved it to app.listen